go/raindrops: analyzer comments (#1216)
exercism/website-copy: automated-comments/go/raindrops/plus_equal_comment.md

> Strings in Go can be appended with a syntax like this: `speech += "Plang"`.

Reviewer: ```suggestion
Strings in Go can be appended using syntax like this: `speech += "Plang"`.
```
